src/main/java/frc/core/util/PID/PIDTalon.java

```diff
@@ -66,12 +67,10 @@ public void setFollowers(TalonSRX... followers) {
 	}
 
 	public void setFollowersInverted(Boolean... isInvertedList) {
-		int index = 0;
 		if (isInvertedList.length >= this.followers.size()) {
-			for (TalonSRX follower : followers) {
-				follower.setInverted(isInvertedList[index]);
-				index++;
-			}
+			followers.forEach(For.withCounter((i, follower) -> {
```

Comment:
bruno, ficou um pouco repetido se tu for pensar em ler essa linha... tipo, parece que tu tá fazendo um for each dentro de um for each...

https://www.baeldung.com/java-foreach-counter#a-functional-for-each

da uma olhada nesse exemplo, é mais ou menos oq tu fez, mas acho que fica mais claro e sem redundância